### ISSUE TITLE
Location info not exposed with parseFragment

### ISSUE THREAD

**Issue by parshap** [inikulin/parse5#82]
It appears that using `parse5.parseFragment()` does not expose the `__location` property on nodes like when using `parse5.parse()`. See below example (using `parse5@2.0.0`):

```js
> var parse5 = require("parse5")
> var html = "<html><head></head><body>foo</body></html>";
> var options = { locationInfo: true };
> parse5.parse(html, options).childNodes[0].__location
{ line: 1,
  col: 1,
  startOffset: 0,
  endOffset: 42,
  startTag: { line: 1, col: 1, startOffset: 0, endOffset: 6 },
  endTag: { line: 1, col: 36, startOffset: 35, endOffset: 42 } }
> parse5.parseFragment(html, options).childNodes[0].__location
undefined
```

**Comment by inikulin** [comment]
Thank you for discovering this!

**Comment by inikulin** [comment]
Published as v2.0.1

**Comment by parshap** [comment]
Excellent! Thank you.
